network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -652,6 +657,13 @@ def get_voting_json(self):
         ),
     ]
 
+    override_translatable_fields = [
+        SynchronizedField('slug'),
+        SynchronizedField('product_url'),
+        SynchronizedField('creepiness_value'),
+        SynchronizedField('votes'),
```

Comment:
Does this mean someone could change the vote values?

Reply:
Yep, that was an accident. I removed that one in the latest commit.

Reply:
Nice, but I guess you will want to remove `creepiness_value` too?

Reply:
Oh yes thank you I missed that this morning! Cheers!